### PR TITLE
Dragonboard 410c AOSP Documentation updated

### PR DESCRIPTION
**Pull request by ric96**
- Updated the documentation to use kernel from @robherring 's branch qcom-4.14
- Updated flashing and building instruction with regards for the same
- Other misc changes
Signed-off-by: Sahaj Sarup <sahaj.sarup@linaro.org>

Tagging for review
@ndechesne

### CODE REVIEW

File: ConsumerEdition/DragonBoard-410c/Guides/AOSP.md [hunk 1]
```diff
@@ -5,63 +5,78 @@ redirect_from: /db410c-getting-started/Guides/AOSP.md/
...
-    cd <ANDROID_TOP>
-    fastboot flash boot out/target/product/db410c/boot.img
-    fastboot flash system out/target/product/db410c/system.img
-    fastboot flash cache out/target/product/db410c/cache.img
-    fastboot flash userdata out/target/product/db410c/userdata.img
-    fastboot reboot
-
+```shell
+cd ..
+cd <KERNEL_TOP>
```
Comment (ndechesne) [inline ConsumerEdition/DragonBoard-410c/Guides/AOSP.md]: this does not look right. you either need to create the boot image in ANDROID_TOP/out/target/... or in KERNEL_TOP/bootxx.img

**Comment by ndechesne** [comment]
there is a 'merge' commit in your PR. you might want to rebase.

File: ConsumerEdition/DragonBoard-410c/Guides/AOSP.md (hunk 1 above)
Comment (ric96) [reply ConsumerEdition/DragonBoard-410c/Guides/AOSP.md]: doh! just a sec

**Comment by ndechesne** [comment]
thanks, i merged it. do you think you can work on building the firmware package from AOSP, so that we can get rid of the googledrive link?

**Comment by ric96** [comment]
@ndechesne yeah

**Pull request merged**